app/models/form/sales/questions/owning_organisation_id.rb

```diff
@@ -93,5 +93,5 @@ def merged_organisation_label(name, merge_date)
     "#{name} (inactive as of #{merge_date.to_fs(:govuk_date)})"
   end
 
-  QUESTION_NUMBER_FROM_YEAR = { 2023 => nil, 2024 => 1, 2025 => 2 }.freeze
+  QUESTION_NUMBER_FROM_YEAR = { 2023 => nil, 2024 => 1 }.freeze
```

Comment:
I think it makes sense to leave in 2025 and set it to 1 instead of removing it. Is there a specific reason you got rid of it completely?

Reply:
🫤
The problem I see with it is that then next year if the question order changes we'll either add it in again (for 0 net benefit) or forget to and break this year's form. I don't see what benefit we get from removing it.

Reply:
ah makes sense 👍, I've changed this now